packages/dashboard/src/store/actions/collapsePane/index.ts

```diff
@@ -0,0 +1,45 @@
+import type { Action } from 'redux';
```

Comment:
general question, do you need this global state ? is it used anywhere else except one component ? if not, i dont think you need this , you can just use local state

Reply:
+1 on this, I think we should instead make a collapsible pane which keeps this state internally.